### ISSUE TITLE
standalone mode FiberError

### ISSUE THREAD

**Issue by machine-w** [websocket-rails/websocket-rails#231]
hi,I have same error as https://github.com/websocket-rails/websocket-rails/issues/191 

if standalone mode = false there are no errors.  

 if I add "config.redis_options = {driver: :ruby}" to config file, rails server will not do work and I  can't use 'ctrl+c' to stop it.

any idea? thanks

**Comment by Envek** [comment]
@inventionlabsSydney thanks a lot, your current code from https://github.com/inventionlabsSydney/websocket-rails helped us (we're switched to standalone mode recently).

@moaa @DanKnox I really happy to know that websocket-rails is evolving. I've tried to switch to `threadsocket-rails` branch but have problems with channel authorization. Also there is errors when I try to render a json template to send via websockets (NoMethod errors). Is there any upgrade guide? May be I missing something…

**Comment by Envek** [comment]
No it didn't really helped us (just for one time on one machine). Currently switched to unicorn as main server and thin as standalone websockets server. This combination works.

**Comment by inventionlabsSydney** [comment]
Hi @Envek,
Unfortunately I haven't had time to look at this as I've been under the pump with my work :(
Hopefully sometime I will be able to help address this. Sorry!

**Comment by shivakumaarmgs** [comment]
@Envek 
> Currently switched to unicorn as main server and thin as standalone websockets server. This
> combination works.

Yeah it does. I had this Fiber problem in my development machine which uses thin as application server, but its running fine in staging environment where I use unicorn as the application server. So after switching to unicorn in development machine, the issue is solved. But I can't understand what is happening? Why is there an error while using Thin? Why is not there when using Unicorn?

**Comment by ajbraus** [comment]
Is there a solution for this issue? This is currently breaking my production :(

**Comment by Karthik-Y** [comment]
@ajbraus it is the faye websocket failing to initialise websocket driver if you got version 0.10.1 then I suggest you to just add start_driver in lib/faye/websocket.rb at line 77 right at the end of the initialize function. Or downgrade your faye websocket gem to one level.

**Comment by synthead** [comment]
I am experiencing this issue as well.

**Comment by Draeken** [comment]
When using 
`gem 'websocket-rails', github: 'moaa/websocket-rails', branch: 'threadsocket-rails'`
`gem 'websocket-rails-js', github: 'websocket-rails/websocket-rails-js', branch: 'sub_protocols'`
`faye-websocket (0.10.0)`

I have this issue:
`'ws://localhost:3000/websocket' failed: Error during WebSocket handshake: Unexpected response code: 200`

**Comment by mathiasxx** [comment]
Turning off eager loading "fixed" the redis crash on boot issue.

```
# in environments/prodution.rb
config.eager_load = false
```
